combat-trainer.lic

```diff
@@ -1779,6 +1782,32 @@ class TrainerProcess
       bput(%w[weave bob circle].sample, 'roundtime', 'There is nothing else', 'Face what', 'You must be closer', 'You must be standing', 'Strangely, you don\'t feel like fighting right now', 'flying too high for you to attack') unless game_state.npcs.empty?
     when 'Analyze'
       analyze(game_state, 0)
+    when 'Locks'
+      return if game_state.loaded
+      return if left_hand
+      boxes = get_boxes(@pet_box_source)
+      return if boxes.length == 0
+      box = boxes.first
+      # Check what's in the right hand so we can get it back
+      rhitem = @equipment_manager.item_by_desc(right_hand) if right_hand
```

Comment:
❓ Wouldn't the `rhitem` normally be a weapon? We have a method on `game_state` that can 'get rid' of the weapon safely (and one that gets it back).